### PR TITLE
drivers/sensor: lis2dw12: add wakeup_duration support

### PR DESCRIPTION
**Pull request by awojak**
Add wakeup_duration support. (WAKE_DUR in WAKE_UP_DUR).
Value is configurable through DT per instance.

### CODE REVIEW

File: dts/bindings/sensor/st,lis2dw12-common.yaml [hunk 1]
```diff
@@ -246,3 +246,19 @@ properties:
       - 5
       - 6
       - 7
+
+  wakeup-duration:
+    type: int
+    default: 0
```
Comment (MaureenHelm) [inline dts/bindings/sensor/st,lis2dw12-common.yaml:252]: Need to justify the default in the description. Is it a register reset value?

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

File: dts/bindings/sensor/st,lis2dw12-common.yaml (hunk 1 above)
Comment (awojak) [reply dts/bindings/sensor/st,lis2dw12-common.yaml:252]: Default value is a register reset value. Add this information to the description ?

Comment (avisconti) [reply dts/bindings/sensor/st,lis2dw12-common.yaml:252]: Yes, correct. Please add it in the description ("default value is the register reset value")

**Review by avisconti — COMMENTED** [review]
LGTM
Just add the description in dts file as requested

**Review by avisconti — APPROVED** [review]

**Comment by awojak** [comment]
@MaureenHelm I made the changes you asked for, can you aprove the Pull Request?

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**